server/command/command.go

```diff
@@ -1062,6 +1068,39 @@ func (r *Runner) actionTest(args []string) {
 	}
 }
 
+func (r *Runner) actionTestGeneratePlaybooks(params []string) {
+	if len(params) < 1 {
+		r.postCommandResponse("The command expects one parameter: <numPlaybooks>")
+		return
+	}
+
+	numPlaybooks, err := strconv.Atoi(params[0])
+	if err != nil {
+		r.postCommandResponse("Error parsing the first argument. Must be a number.")
+		return
+	}
+
+	if numPlaybooks > 5 {
+		r.postCommandResponse("Cant generate playbooks since the number of playbook generated must be at most five")
```

Comment:
We can reword this to something more simple:

```suggestion
		r.postCommandResponse("Maximum number of playbooks is 5")
```